src/ui/handler/scroll_zoom.js

```diff
@@ -129,16 +133,21 @@ class ScrollZoomHandler {
      *
      * @param {Object} [options] Options object.
      * @param {string} [options.around] If "center" is passed, map will zoom around center of map.
+     * @param {boolean} [options.requireCtrl] If set to true, ctrl (or ⌘ in OS devices) must be pressed while scrolling to zoom.
```

Comment:
I think this should be fine. I've previously confirmed that HTML entities work as well, e.g. `&#8984;`: &#8984;, just in case unicode is an issue anywhere between here and the docs, though I think it should be just fine.